x/curating/keeper/querier.go

```diff
@@ -37,5 +38,43 @@ func queryParams(ctx sdk.Context, k Keeper) ([]byte, error) {
 	return res, nil
 }
 
-// TODO: Add the modules query functions
-// They will be similar to the above one: queryParams()
+func queryPost(ctx sdk.Context, path []string, req abci.RequestQuery, k Keeper) ([]byte, error) {
+	i64, _ := strconv.ParseUint(path[0], 10, 32)
+	vendorID := uint32(i64)
+	postID := path[1]
+
+	post, found, err := k.GetPost(ctx, vendorID, postID)
+	if err != nil || !found {
+		return nil, types.ErrPostNotFound
```

Comment:
@shanev, I'm returning the error here, but unfortunately when run on the CLI I don't get the error message... just `internal error`.